static/openapi/p3-spec.json

```diff
@@ -0,0 +1,3479 @@
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "400": {
+              "description": "Bad request. Possible issue with the request body."
+            },
+            "500": {
+              "description": "Internal server error."
+            }
+          }
+        }
+      },
+      "/api/p3/v1/ppm/jobs/{jobId}": {
+        "delete": {
+          "tags": [
+            "PPM"
+          ],
+          "summary": "Delete a job",
+          "description": "Deletes a specific PPM job based on the job ID provided.",
+          "security": [
+            {
+              "ApiKeyAuth": []
+            }
+          ],
+          "parameters": [
+            {
+              "in": "path",
+              "name": "jobId",
+              "required": true,
+              "schema": {
+                "type": "string",
+                "example": "13123-13123-31231-ddddd"
+              },
+              "description": "The unique ID of the job to delete."
+            }
+          ],
+          "responses": {
+            "204": {
+              "description": "No Content - Successfully deleted the job."
+            },
+            "400": {
+              "description": "Bad request. Possible issue with the job key provided."
+            },
+            "404": {
+              "description": "The specified job key does not exist."
+            },
+            "500": {
+              "description": "Internal server error."
+            }
+          }
+        }
+      },
+      "/api/p3/v1/auth/apikeys": {
+        "post": {
+          "summary": "Create API key",
+          "tags": [
+            "Authentication"
+          ],
+          "security": [
+            {
+              "ApiKeyAuth": []
+            }
+          ],
+          "description": "Create API key",
+          "operationId": "createApiKey",
+          "requestBody": {
+            "required": true,
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ApiKeyCreateRequest"
+                }
+              }
+            }
+          },
+          "responses": {
+            "201": {
+              "description": "Successfully created API key.",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "$ref": "#/components/schemas/ApiKeyCreateResponse"
+                  }
+                }
+              }
+            },
+            "400": {
+              "description": "Bad Request."
+            },
+            "401": {
+              "description": "Authorization failure due to invalid authentication credentials."
+            },
+            "403": {
+              "description": "Either the API key doesn't have permissions or it is deactivated."
+            },
+            "409": {
+              "description": "Duplicate API key."
```

Comment:
Wait, what? The API keys are unique. Do you mean that one already exists with the same name?